### DISCUSSION TITLE
Pursa slicer 2.6 fetures in superslicer when?

### DISCUSSION THREAD

**Discussion by ghost** [supermerill/SuperSlicer#3621] [General]
not to rush the dev or anyone 

is there or going to be pursa slicers 2.6 features in superslicer or is this repo dead

**Comment by jhstatewide** [comment]
It's paused for a bit, re: 2.6 your guess is as good as anyone. It's up to supermerill or anyone else with the necessary motivation / skills to do the job.

**Comment by levleon13** [comment]
variable layer height + trees is the only thing I need to make this slicer perfect. 
What's the status of the dev? What does a pause mean?

**Reply by DVSVIDEO** [reply]
The variable layer height is already included. Didn't you notice?

**Comment by jhstatewide** [comment]
Superslicer is largely developed by a single person who is not currently working on it. On Feb 4, 2023 4:27 AM, levleon13 ***@***.***> wrote:
variable layer height + trees is the only thing I need to make this slicer perfect.
What's the status of the dev? What does a pause mean?

**Reply by levleon13** [reply]
in that case, i guess sponsoring has no point.

**Comment by jhstatewide** [comment]
Sure it does. if you've ever made use of superslicer in the past it is gratitude, or it may encourage the Dev to resume work soon. I don't know him personally but he has made a commit or two in the past couple of weeks so he definitely is still around. On Feb 4, 2023 8:08 AM, levleon13 ***@***.***> wrote:
in that case, i guess sponsoring has no point.

**Reply by CD1010** [reply]
Well, maybe he is waiting for Prusa to release 2.6.  That would makes sense, why merge in alpha code to start with.  Wait till release.

**Reply by Contrasens** [reply]
The development seem stopped to me....:( Not just the releases are stalling, but also the open issues are accumulating without anyone handling them.

**Comment by Annuit20** [comment]
That would be pretty sad. I love "my" Superslicer.  
A liddle donation is on the way. I hope some follow :)

**Reply by CD1010** [reply]
Super slicer kicks ass. Print time on my corexy same model superslicer generates print in half time i can get prusa slicer to do it on same corexy printer.  Not sure why.  But wish he had new features they added.
